src/uu/comm/src/comm.rs

```diff
@@ -96,26 +129,23 @@ fn comm(a: &mut LineReader, b: &mut LineReader, opts: &ArgMatches) {
         match ord {
             Ordering::Less => {
                 if !opts.get_flag(options::COLUMN_1) {
-                    ensure_nl(ra);
-                    print!("{ra}");
+                    print!("{}", String::from_utf8_lossy(ra));
```

Comment:
Maybe we should just write the bytes here? I.e.
```rust
let out = std::io::stdout().lock();
out.write_all(ra)?;
```
Although a downside might be that it will break on non-utf8 on Windows?

Reply:
Yes, non-utf8 byte sequences are not supported by `Stdout` on Windows. Another point is consistency: the other arms use `print!`, too.

Reply:
Re: consistency. It would indeed be best to change all arms. But I agree that it doesn't make sense for this PR at least.